addons/popochiu/migration/do_migration.gd

```diff
@@ -0,0 +1,97 @@
+@tool
+class_name DoMigration
```

Comment:
I like `MigrationsRunner` and `MigrationsManager`. Maybe the second one covers more taking into account this is the class that will be in charge of dealing with the migrations.

To be honest, I kept the name (until today) because it was the one used by Anthony. I'll make the change.